packages/aws-cdk-lib/aws-appsync/lib/graphqlapi.ts

```diff
@@ -621,6 +630,9 @@ export class GraphqlApi extends GraphqlApiBase {
     if (props.resolverCountLimit !== undefined && (props.resolverCountLimit < 0 || props.resolverCountLimit > 10000)) {
       throw new Error('You must specify a resolver count limit between 0 and 10000.');
     }
+    if (props.ownerContact !== undefined && (props.ownerContact.length > 256)) {
+      throw new Error('You must specify a owner contact as a string of 256 characters or less.');
```

Comment:
It would be more helpful to align it with the name of the property.

```suggestion
      throw new Error('You must specify `ownerContact` as a string of 256 characters or less.');
```